### PR TITLE
Add .md creation to autoTester

### PR DESCRIPTION
**Pull request by NissimHadar**
AutoCreator can now create a .md file from the test.js script in a test folder.
In addition, the access to raw GitHub files has been changed.

## TEST PLAN
This is qa ed by engine team as we are developing autotester as a n internal tool

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8455/HighFidelity-Beta-PR12771-cb5bd4c0804d8ac1e02ad7e6ca0feeb5c76b328e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8455/HighFidelity-Beta-PR12771-cb5bd4c0804d8ac1e02ad7e6ca0feeb5c76b328e.exe

File: tools/auto-tester/src/Test.h [hunk 1]
```diff
@@ -47,7 +65,7 @@ class Test {
     void zipAndDeleteTestResultsFolder();
 
     bool isAValidDirectory(QString pathname);
-
+	QString extractPathFromTestsDown(QString fullPath);
```
Comment (samcake) [inline tools/auto-tester/src/Test.h]: bad spacing

**Comment by samcake** [comment]
You have a warning on linux

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8488/HighFidelity-Beta-PR12771-7a6b071e46d7e9e1c70618accff97b69de305c9e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8488/HighFidelity-Beta-PR12771-7a6b071e46d7e9e1c70618accff97b69de305c9e.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8492/HighFidelity-Beta-PR12771-bb0d4a66b6f537b44530a3f90420d981ad4edc9b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8497/HighFidelity-Beta-PR12771-235996911a64002c94448a8f3f0d46295ee91cfe.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8497/HighFidelity-Beta-PR12771-235996911a64002c94448a8f3f0d46295ee91cfe.dmg

**Comment by samcake** [comment]
Approved

**Pull request merged**